solr/contrib/ltr/src/java/org/apache/solr/search/LTRQParserPlugin.java

```diff
@@ -151,7 +159,7 @@ public LTRQParser(String qstr, SolrParams localParams, SolrParams params,
     @Override
     public Query parse() throws SyntaxError {
       // ReRanking Model
-      final String modelName = localParams.get(CommonLTRParams.MODEL);
+      final String modelName = localParams.get(LTRFeatureLoggerTransformerFactory.MODEL);
```

Comment:
question: What about having MODEL declared here and then reused in LTRFeatureLoggerTransformerFactory? during the query processing this class comes before so it would make sense for me to have it here...